docs/content/preview/yugabyte-cloud/cloud-clusters/backup-clusters.md

```diff
@@ -117,3 +117,69 @@ To restore a backup of a cluster:
 1. Choose the databases or keyspaces to restore and click **Next**.
 1. Select the target cluster.
 1. Click **Restore Now**.
+
+## Remote backup replication
+
+Use remote backup replication to copy all your cluster backups (scheduled, incremental, and on demand) to a storage bucket in the same cloud provider.
+
+Only new backups (that is, backups created after backup replication has been enabled on the cluster) are transferred. Backups are transferred to the bucket once a day.
+
+Remote backup replication counts against your data transfer allowance. This may incur additional costs for network transfer, especially for cross-region transfers, if usage exceeds your cluster allowance. To avoid cross-region data transfer costs, use a bucket in the same region as the cluster.
+
+To restore from a remote backup, contact {{% support-cloud %}}.
+
+Currently, only clusters deployed to GCP are supported.
+
+### Prerequisites
+
+The remote storage bucket must be on the same cloud provider as the cluster; for clusters deployed in GCP, the bucket must be in Google Cloud Storage. Transfer is performed using [Google Storage Transfer Service](https://cloud.google.com/storage-transfer/docs/overview).
+
+#### Set up target buckets
+
+1. Identify and configure the Google Cloud Storage buckets that will serve as replication targets for your backup data.
+
+    It is highly recommended to use a new, empty bucket rather than one that already contains data. Although safeguards are in place, remote replication requires delete permissions. To minimize the risk of accidental data loss, using a fresh bucket is the safest approach.
+
+2. Set up service account permissions.
+
+    Grant the required permissions to the Storage Transfer Service agent on each target bucket.
+
+    Service Account Principal:
```

Comment:
looks like the service account principal is an email ID for the transfer agent? Hoping the details are in the links you added about it